.circleci/config.yml

```diff
@@ -38,4 +38,4 @@ jobs:
           key: v1-dependencies-{{ checksum "pom.xml" }}
 
       # run tests!
-      - run: mvn clean package
+      - run: mvn clean install
```

Comment:
for tests it can be `mvn clean verify`

Reply:
Yes, the ci build was failing because the project had to be built before the test phase to have all the necessary modules. Now it's working.